src/CalculatorUITestFramework/CalcHistoryFlyout.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+using System.Collections.ObjectModel;
+using OpenQA.Selenium.Interactions;
+using System.Windows.Input;
+
+namespace CalculatorUITestFramework
+{
+    public class CalcHistoryFlyout
+    {
+        private WindowsDriver<WindowsElement> session => WinAppDriver.Instance.CalculatorSession;
+        public HistoryPanel HistoryPanel = new HistoryPanel();
+        public WindowsElement Header => this.session.TryFindElementByAccessibilityId("Header");
+        public WindowsElement AppName => this.session.TryFindElementByAccessibilityId("AppName");
+        public WindowsElement HistoryFlyout => this.session.TryFindElementByAccessibilityId("HistoryFlyout");
+        public AppiumWebElement ListViewItem => WinAppDriver.Instance.CalculatorSession.FindElementByClassName("Popup").FindElementByClassName("ListViewItem");
+        public AppiumWebElement HistoryListView => WinAppDriver.Instance.CalculatorSession.FindElementByClassName("Popup").FindElementByAccessibilityId("HistoryFlyout").FindElementByAccessibilityId("HistoryListView");
+        public AppiumWebElement ClearHistoryButton => WinAppDriver.Instance.CalculatorSession.FindElementByClassName("Popup").FindElementByAccessibilityId("HistoryFlyout").FindElementByAccessibilityId("ClearHistory");
+        public AppiumWebElement HistoryEmpty => WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("HistoryFlyout").FindElementByAccessibilityId("HistoryEmpty");
+        public WindowsElement ExprTextBlock => this.session.TryFindElementByAccessibilityId("ExprTextBlock");
+        public WindowsElement ResultTextBlock => this.session.TryFindElementByAccessibilityId("ResultTextBlock");
+
+
+        /// <summary>
+        /// Opens the History Flyout.
+        /// </summary>
+        public void OpenHistoryFlyout()
+        {
+            this.HistoryPanel.ResizeWindowToNotDiplayHistoryLabel();
+            this.AppName.Click();
+            this.Header.SendKeys(Keys.Control + "h" + Keys.Control);
```

Comment:
>h [](start = 49, length = 1)

I think alt i is the command to open the history pane.  "h" opens the calc mode picker flyout.

Reply:
- Pressing [Ctrl] + [H] open the history flyout panel when the calculator is to small to display the history panel
- Pressing [Alt] + [H] opens the navigation menu